bitwarden_license/src/Portal/EnterprisePortalCurrentContext.cs

```diff
@@ -52,7 +52,8 @@ public async override Task SetContextAsync(ClaimsPrincipal user)
 
             // TODO: maybe make loading orgs Lazy<T> somehow?
             var orgUserRepo = _serviceProvider.GetRequiredService<IOrganizationUserRepository>();
-            var userOrgs = await orgUserRepo.GetManyDetailsByUserAsync(UserId.Value);
+            var userOrgs = await orgUserRepo.GetManyDetailsByUserAsync(UserId.Value,
+                Core.Enums.OrganizationUserStatusType.Confirmed);
```

Comment:
Any reason to not just import to clean up this param?

Reply:
no real reason. Generally it makes sens to import if you are doing things more than once.

Reply:
Just curious! Sounds good.